Session/Media Viewing & Editing/MessageInfoScreen.swift

```diff
@@ -305,7 +306,10 @@ struct MessageInfoScreen: View {
                                     Button(
                                         action: {
                                             actions[index].work()
-                                            dismiss()
+                                            feedbackMessage = actions[index].feedback
+                                            DispatchQueue.main.asyncAfter(deadline: .now() + 2, execute: {
+                                                dismiss()
```

Comment:
I only just realised that we dismiss the screen after triggering one of the actions - just wondering whether that's the desired behaviour for all of the actions? (eg. `Copy`, `Save` and `Copy Account ID` probably shouldn't automatically dismiss the screen)

Reply:
Yeah, it is odd to dismiss to screen after tapping one of the actions. I didn't find any reference on this. I think we can remove the dismiss logic.